content/release-notes/2022-10-07.md

```diff
@@ -0,0 +1,13 @@
+---
+label: 'Storage'
+---
+
+### What's new
+
+- Pageserver: Increase default compaction_period setting to 20 s
+- Pageserver: Initial support for online tenant rebalancing
+- Pageserver: Support for multiple postgres versions (14 and 15 for now)
+- Compute: Add `h3_pg` extension
+- Compute: Beta version of sequential scan page prefetch
+- Compute: Backpressure algorithm tuning
```

Comment:
```suggestion
- Compute: Tuned the backpressure algorithm, which manages the lag between the Pageserver and compute node or the Pageserver and the Write-ahead Log (WAL) service.
```

Reply:
should we add 'backpressure' to glossary?
what are the symptoms that this particular adjustment improves or relieves?

Reply:
Created a PR for "backpressure" glossary addition.
Updated the release note entry, which was reviewed by @knizhnik.